Gemfile.lock

```diff
@@ -273,6 +273,7 @@ DEPENDENCIES
   dotenv-rails
   font-awesome-sass (~> 5.6.1)
   jbuilder (~> 2.0)
+  kaminari
```

Comment:
I thought you deleted it ?

Reply:
i did it, but it's still in the gemfile.lock dependcies, not sure if i can delete it